JavaScript/Github_unfollowers/README.md

```diff
@@ -0,0 +1,18 @@
+# Github_Unfollowers
+This is a script to track recent unfollowers on github. 
+* This script uses the github API to fetch data of followers.
+* The data is then stored in old.json file and compared with new data and hence the unfollowers can be found!
```

Comment:
Use backticks for `old.json` file. If possible you can replace this with a more friendly or understanding filename, though that's entirely optional